### PR TITLE
Fix for #33: do not use loopback IP addresses

### PR DESCRIPTION
**Pull request by JochenHiller**
As described in #33: do not use loopback IP addresses

### CODE REVIEW

File: network.go [hunk 1]
```diff
@@ -67,6 +67,11 @@ func localIPv4MCastAddrs() ([]string, error) {
 				// Not IPv4.
 				continue
 			}
+			// Fix #33: consider NOT loopback IP addresses
+			if addr.IP.IsLoopback() {
```
Comment (huin) [inline network.go:71]: Could you test if checking discarding interfaces for which `iface.Flags&net.FlagLoopback != 0`, a few lines earlier?

File: network.go [hunk 2]
```diff
@@ -67,6 +67,11 @@ func localIPv4MCastAddrs() ([]string, error) {
 				// Not IPv4.
 				continue
 			}
+			// Fix #33: consider NOT loopback IP addresses
```
Comment (huin) [inline network.go:70]: No need to add comments addressing bugs. That's effectively in the version history and can distract from the code.

File: network.go (hunk 1 above)
Comment (huin) [reply network.go:71]: (This would also move the check earlier in this function)

Comment (JochenHiller) [reply network.go]: Yep, did work. Thanks for hint, is more effective.

File: network.go (hunk 2 above)
Comment (JochenHiller) [reply network.go]: OK got it. Removed ref to issue

**Comment by huin** [comment]
Looks good, thanks for your contribution!

**Pull request merged**